### DISCUSSION TITLE
Allow people who have not logged in to enable dark mode

### DISCUSSION THREAD

**Discussion by FrostBird347** [community/community#22168] [Accessibility]
<p>There does not appear to be any way to enable dark mode without being logged in.</p>

**Comment by FranklinYu** [comment, marked as answer]
<p>According to <a href="https://github.community/t/make-dark-mode-a-browser-setting-instead-of-a-user-setting/148410/5" class="inline-onebox">Make Dark Mode a browser setting instead of a user setting</a>, this will be default when it’s out of beta.</p>

**Comment by cehfisher** [comment]
Hi @FranklinYu - this issue seems to have been resolved. Could you please confirm and close this issue? Thanks! (cc @queenofcorgis)

**Reply by FrostBird347** [reply]
I have closed the issue
